gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -2,6 +2,7 @@
 
 import argparse
 import time
+import os
```

Comment:
should be in alphabetical order `os < time`

Reply:
addressed in a new commit